charts/cluster-autoscaler/Chart.yaml

```diff
@@ -17,4 +17,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.1.0
+version: 9.1.1
```

Comment:
I'd prefer if we stuck to SemVer here and bumped the minor version as new functionality is being added